packages/common/src/constants/EngineSettings.ts

```diff
@@ -37,6 +37,15 @@ export const EngineSettings = {
     ApiKey: 'apiKey',
     Url: 'url'
   },
+  Metabase: {
+    SiteUrl: 'siteUrl',
+    SecretKey: 'secretKey',
+    CrashDashboardId: 'crashDashboardId',
+    Expiration: 'expiration',
+    ProjectDashboardId: 'projectDashboardId',
+    AccountDashboardId: 'accountDashboardId',
```

Comment:
This doesnt needs to be in core engine as its MT specific.

Reply:
Removed